docker-compose.azure.yml

```diff
@@ -0,0 +1,27 @@
+version: "3.8"
+
+services:
+    ibf-api-service:
+        image: gulfaraz/ibf-api-service
```

Comment:
Should we create a "rodekruis"-account on DockerHub for this?
So that we're not depending on a personal account for this.

Reply:
updated to `rodekruis` images.. I've sent team invites on docker hub for access..